src/lib/cht-api.ts

```diff
@@ -182,20 +189,28 @@ export class ChtApi {
       include_docs: true,
     };
     console.log('axios.get', url, params);
-    const resp = await this.axiosInstance.get(url, { params });
+    let places: any[] | undefined = ChtApi.cache.get(placeType);
+    if (places === undefined) {
+      const resp = await this.axiosInstance.get(url, { params });
+      places = resp.data.rows;
+      ChtApi.cache.set(placeType, places);
+    }
 
-    return resp.data.rows
-      .map((row: any): RemotePlace => {
+    return places?.map((row: any): RemotePlace => {
```

Comment:
welcome you to implement the cache via the `remote-place-cache` class but using this node-cache library.  everything in ChtApi isn't easily unit testable, but having this cache in a separate class allows it to be continue to be tested